.dockerignore

```diff
@@ -0,0 +1,4 @@
+.idea
+*.class
+build
+*.iml
```

Comment:
it is just a common practice, files should end with an empty new line

Reply:
ok cool